src/mbyte.c

```diff
@@ -5644,6 +5644,8 @@ f_setcellwidths(typval_T *argvars, typval_T *rettv UNUSED)
 	// Clearing the table.
 	VIM_CLEAR(cw_table);
 	cw_table_size = 0;
+	changed_window_setting_all();
+	redraw_all_later(UPD_CLEAR);
 	return;
```

Comment:
Will it be better to add a `done:` label at the bottom and use a `goto done;` here?

Reply:
OK, updated.